blitzstudios/sleeper-mini-core
# Add Logging back from Sleeper app

### Description
This change will allow console.log messages to be sent back to the mini's packager to assist with debugging.

Also fixed a small bug with Windows Docker files not automatically recompiling after changes.

### How To Test

1. Add console.log() to a mini
2. See output when testing minis through development connection.
